vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -694,4 +694,15 @@
     <Error Condition="'$(Source)'==''" Text="To generate a pre-processed file, please specify a source file with msbuild /p:Source=nameOfTheSource.cpp" />
     <CL PreprocessToFile="true" Sources="$(Source)" AdditionalIncludeDirectories="%(ClCompile.AdditionalIncludeDirectories)" PreprocessorDefinitions="%(ClCompile.PreprocessorDefinitions)" AdditionalOptions="%(ClCompile.AdditionalOptions) /d1PP" PreprocessKeepComments="true" PreprocessOutputPath="$(Source.Replace('.cpp', '.pp'))" />
   </Target>
+
+  <!-- https://github.com/microsoft/microsoft-ui-xaml/issues/3133 -->
+  <Target Name="InjecForcedXamlIncludes" BeforeTargets="CompileXamlGeneratedFiles" >
```

Comment:
I don't think this will be run. This would run before CompileXamlGeneratedFiles _if_ told to run or required as a prerequisite by a target told to run, but in and of itself, it isn't required to run

Reply:
Some more detail in a Teams thread, but this seems to be run when looking at log outputs with diagnostic verbosity. Could totally be missing something here though since MSBuild is not my forte

Reply:
The BeforeTargets and AfterTargets are reverse-association, so if `CompileXamlGeneratedFiles` is run, it will run `InjecForcedXamlIncludes` before it, there needs to be no declaration for DependsOnTargets for that target to make it run.